ibm/mas_devops/roles/cp4d_service/defaults/main.yml

```diff
@@ -66,7 +66,7 @@ cpd_wsl_project_storage_guid: "{{ 1000 | random | to_uuid | lower }}"
 # Note: This is only used when mas_instance_id and mas_config_dir are not empty string
 cpd_default_wd_instance_name: "wd-mas-{{ mas_instance_id }}-assist"
 cpd_wd_instance_name: "{{ lookup('env', 'CPD_WD_INSTANCE_NAME') | default(cpd_default_wd_instance_name, true) }}"
-cpd_wd_deployment_type: "{{ lookup('env', 'CPD_WD_DEPLOYMENT_TYPE') | default('Starter', true) }}" # Default is "Starter" which creates 1 replica of each pod, "Production" creates at least 2 replicas of each pod, this cannot be changed in the future, and if you upgrade wd, you should use same type.
+cpd_wd_deployment_type: "{{ lookup('env', 'CPD_WD_DEPLOYMENT_TYPE') | default('Production', true) }}" # creates 1 replica of each pod if you use Starter, It is intended for demonstration and trial purposes only. Default is "Production" which creates at least 2 replicas of each pod, this cannot be changed in the future, and if you upgrade wd, you should use same type.
```

Comment:
@liyongc have you tested the install for WD with this type of deployment? Just asking because as you know, we have found all sorts of issues installing CPD services and we really want to make sure nothing disruptive happens when we change something like this.